### PR TITLE
Adding the ability to disable interpolation in logs

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

File: tests/Database/Unit/Driver/AbstractDriverTest.php [hunk 1]
```diff
@@ -89,6 +92,64 @@ public function testClone(): void
...
+            ->willReturn($this->createMock(PDOStatementInterface::class));
+
+        $driver = TestDriver::createWith(
+            new SQLiteDriverConfig(options: ['enableInterpolationInLogs' => true]),
+            $this->createMock(HandlerInterface::class),
+            $this->createMock(BuilderInterface::class),
+            $pdo
+        );
+
+        $logger = new TestLogger();
```
Comment (roxblnfk) [inline tests/Database/Unit/Driver/AbstractDriverTest.php]: Could you use mock instead?

File: src/Config/DriverConfig.php [hunk 2]
```diff
@@ -26,6 +26,7 @@ abstract class DriverConfig
 
     protected array $defaultOptions = [
         'withDatetimeMicroseconds' => false,
+        'enableInterpolationInLogs' => false,
```
Comment (roxblnfk) [inline src/Config/DriverConfig.php]:
```suggestion
        'logQueryParameters' => false,
```

**Review by roxblnfk — APPROVED** [review]

File: tests/Database/Unit/Driver/AbstractDriverTest.php (hunk 1 above)
Comment (msmakouz) [reply tests/Database/Unit/Driver/AbstractDriverTest.php]: Done

File: src/Config/DriverConfig.php (hunk 2 above)
Comment (msmakouz) [reply src/Config/DriverConfig.php]: Renamed

**Pull request merged**